<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnlitTextureMaterialWidget</class>
 <widget class="QWidget" name="UnlitTextureMaterialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="lblTextureRotation">
     <property name="text">
      <string>Texture rotation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblTextureScale">
     <property name="text">
      <string>Texture scale</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsDoubleSpinBox" name="textureScaleSpinBox">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QgsDoubleSpinBox" name="textureRotationSpinBox">
     <property name="suffix">
      <string> Â°</string>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QgsImageSourceLineEdit" name="textureFile" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsImageSourceLineEdit</class>
   <extends>QWidget</extends>
   <header>qgsfilecontentsourcelineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
